packages/riipen-ui-docs/src/pages/components/typography/typography.md

```diff
@@ -7,15 +7,18 @@ set of type sizes that work well together along with the layout grid.
 
 ## General
 
-The *Roboto* font will **not** be automatically loaded by Riipen-UI.
+The _Roboto_ font will **not** be automatically loaded by Riipen-UI.
```

Comment:
Not sure why this changed, I think an auto-save thing?

Reply:
yea your eslint/prettier is probs formatting stuff on save, all good